cdc/processor/processor.go

```diff
@@ -580,8 +580,8 @@ func (p *processor) handlePosition() error {
 	p.metricCheckpointTsLagGauge.Set(float64(oracle.GetPhysical(time.Now())-checkpointPhyTs) / 1e3)
 	p.metricCheckpointTsGauge.Set(float64(checkpointPhyTs))
 
-	if minResolvedTs > p.changefeed.TaskPosition.ResolvedTs ||
-		minCheckpointTs > p.changefeed.TaskPosition.CheckPointTs {
+	if minResolvedTs != p.changefeed.TaskPosition.ResolvedTs ||
+		minCheckpointTs != p.changefeed.TaskPosition.CheckPointTs {
```

Comment:
What does it mean when `minResolvedTs` is less than `p.changefeed.TaskPosition.ResolvedTs`
or `minCheckpointTs` less than `p.changefeed.TaskPosition.CheckPointTs`

Reply:
added a comment